charts/metering-ansible-operator/templates/crds/reportquery.crd.yaml

```diff
@@ -23,3 +23,103 @@ spec:
   - name: Age
     type: date
     JSONPath: .metadata.creationTimestamp
+
+  validation:
+  openAPIV3Schema:
+    required: ["spec"]
+    type: object
+    properties:
+      spec:
+        type: object
+        required: ["columns", "query"]
+        properties:
+          columns:
+            type: array
+            required: ["names"]
+            items:
+              type: object
+              required: ["type"]
+              properties:
+                name:
+                  type: string
+                  minLength: 1
+                type:
+                  type: string
+                  enum:
+                    - BOOLEAN
+                    - TINYINT
+                    - SMALLINT
+                    - INTEGER
+                    - BIGINT
+                    - REAL
+                    - DOUBLE
+                    - DECIMAL
+                    - VARCHAR
+                    - CHAR
+                    - VARBINARY
+                    - JSON
+                    - DATE
+                    - TIME
+                    - TIMESTAMP
+                    - ARRAY
+                    - MAP
+                    - MAP<VARCHAR, VARCHAR>
+                    - MAP<VARCHAR, INT>
+                    - MAP<INT, INT>
+                    - MAP<INT, VARCHAR>
+                    - ROW
+                    - IPADDRESS
+                    - UUID
+                    - HYPERLOGLOG
+                    - P4HYPERLOGLOG
+                    - QDIGEST
+                unit:
+                  type: string
+                  enum:
+                    - DATE
```

Comment:
I believe these are all lowercase in our default ReportQueries, so the unit enum values need to be lowercase.